eslint.config.js

```diff
@@ -1,15 +1,16 @@
-import globals from 'globals'
-import ts from '@typescript-eslint/eslint-plugin'
-import tsParser from '@typescript-eslint/parser'
-import * as graphqlESLint from '@graphql-eslint/eslint-plugin'
-import eslintJsPlugin from '@eslint/js'
+import globals from "globals";
+import ts from "@typescript-eslint/eslint-plugin";
+import tsParser from "@typescript-eslint/parser";
+import * as graphqlESLint from "@graphql-eslint/eslint-plugin";
+import eslintJsPlugin from "@eslint/js";
+import stylistic from "@stylistic/eslint-plugin";
 
-const gqlSchemaPath = './src/typeDefs/schema.graphql'
+const gqlSchemaPath = "./src/typeDefs/schema.graphql";
```

Comment:
My Prettier keeps changing the single quotes to double in certain files, running ESLint --fix does nor resolve this issue. So that made me think if we need to have a Prettier config file or have a .prettierignore and use the ESLint --fix only (I would go with the last one?)

Reply:
ohhhhh.....we don't use prettier. that's why it's conflicting.

Reply:
So I have Prettier as a plugin for my VS Code, the thing I have read online is to add a prettierignore file to your project? Or is this not correct? 

Because I remember that LaShawn had also something similar with adding double quotes.

Reply:
Ah Sorry Philip, I just had to disable the plugin for the find a doc workspace!